packages/rocketchat-ui/lib/chatMessages.coffee

```diff
@@ -186,6 +186,8 @@ class @ChatMessages
 							else
 								Meteor.call 'slashCommand', {cmd: command, params: param, msg: msgObject }
 							return
+
+					if !RocketChat.settings.get('Message_AllowUnrecognizedSlashCommand')
 						invalidCommandMsg =
```

Comment:
I think you should keep things inside `if match` to keep things working as before